src/react/components/common/assetPreview/videoAsset.tsx

```diff
@@ -73,36 +73,38 @@ export class VideoAsset extends React.Component<IVideoAssetProps> {
                 autoPlay={autoPlay}
                 src={videoPath}>
                 <BigPlayButton position="center" />
-                <ControlBar autoHide={false}>
-                    <CustomVideoPlayerButton order={1.1}
-                        accelerators={["ArrowLeft", "a", "A"]}
-                        tooltip={strings.editorPage.videoPlayer.previousExpectedFrame.tooltip}
-                        onClick={this.movePreviousExpectedFrame}>
-                        <i className="fas fa-caret-left fa-lg" />
-                    </CustomVideoPlayerButton>
-                    <CustomVideoPlayerButton order={1.2}
-                        accelerators={["ArrowRight", "d", "D"]}
-                        tooltip={strings.editorPage.videoPlayer.nextExpectedFrame.tooltip}
-                        onClick={this.moveNextExpectedFrame}>
-                        <i className="fas fa-caret-right fa-lg" />
-                    </CustomVideoPlayerButton>
-                    <CurrentTimeDisplay order={1.3} />
-                    <TimeDivider order={1.4} />
-                    <PlaybackRateMenuButton rates={[5, 2, 1, 0.5, 0.25]} order={7.1} />
-                    <VolumeMenuButton enabled order={7.2} />
-                    <CustomVideoPlayerButton order={8.1}
-                        accelerators={["q", "Q"]}
-                        tooltip={strings.editorPage.videoPlayer.previousTaggedFrame.tooltip}
-                        onClick={this.movePreviousTaggedFrame}>
-                        <i className="fas fa-step-backward"></i>
-                    </CustomVideoPlayerButton>
-                    <CustomVideoPlayerButton order={8.2}
-                        accelerators={["e", "E"]}
-                        tooltip={strings.editorPage.videoPlayer.nextTaggedFrame.tooltip}
-                        onClick={this.moveNextTaggedFrame}>
-                        <i className="fas fa-step-forward"></i>
-                    </CustomVideoPlayerButton>
-                </ControlBar>
+                {autoPlay &&
```

Comment:
will autoPlay ever be false? it's not something user can set right?  When I navigate away from a video and come back, it always autoplay

Reply:
Right now autoPlay is `false` on the sidebar videos and `true` for the main canvas video.